apps/web/components/booking/BookingListItem.tsx

```diff
@@ -323,6 +338,17 @@ function BookingListItem(booking: BookingItemProps) {
         setIsOpenDialog={setIsOpenRescheduleDialog}
         bookingUId={booking.uid}
       />
+      <ReassignDialog
+        isOpenDialog={isOpenReassignDialog}
+        setIsOpenDialog={setIsOpenReassignDialog}
+        bookingId={booking.id}
+        teamId={booking.eventType?.team?.id || 0}
+        assignedHosts={
+          booking.user
+            ? [booking.user.email || ""].concat(booking.attendees.map((attendee) => attendee.email))
+            : []
```

Comment:
I think instead of `booking.user.email` we should use `booking.userPrimaryEmail`